server/src/main/scala/org/dbpedia/extraction/server/resources/TemplateStatistics.scala

```diff
@@ -170,7 +171,7 @@ class   TemplateStatistics(@PathParam("lang") langCode: String, @QueryParam("p")
             var redirectMsg = ""
             for (redirect <- targetRedirect) {
               if (mappingStat.isMapped) {
-                  //redirectMsg = " NOTE: the mapping for " + wikiDecode(redirect, language).substring(createMappingStats.templateNamespace.length()) + " is redundant!"
+                  redirectMsg = " NOTE: the mapping for " + redirect.substring(manager.templateNamespace.length) + " is redundant!"
```

Comment:
I see, makes sense. Then maybe enclose it with a different css class to make it easier for someone to copy the title